BrainPortal/app/views/signups/new.html.erb

```diff
@@ -24,180 +24,167 @@
-                                        "Researcher",
-                                        "Other"
-              ],
-              @signup.position)
-          %>
-      </td>
-    </tr>
-
-    <tr>
-      <th><%= f.label(:affiliation, "* Current affiliation:") %></th>
-      <td><%= f.select :affiliation, options_for_select(
-              [ [ "(select one)", "" ], "Academic",
-                                        "Private sector",
-                                        "Government",
-                                        "Non-profit",
-                                        "Other",
-              ],
-              @signup.affiliation)
-          %>
-      </td>
-    </tr>
-
-    <tr>
-      <th><%= f.label(:email, "* Institutional Email address:") %></th>
-      <td><%= f.text_field :email, :size => 60 %>
-        <br><small>Please supply an address from your research institution</small>
-      </td>
-    </tr>
-
-    <tr>
-      <td colspan="2">&nbsp;</td>
-    </tr>
-
-    <tr>
-      <th><%= f.label(:street1, "Street Address (line 1):") %></th>
-      <td><%= f.text_field :street1, :size => 60 %></td>
-    </tr>
-
-    <tr>
-      <th><%= f.label(:street2, "Street Address (line 2):") %></th>
-      <td><%= f.text_field :street2, :size => 60 %></td>
-    </tr>
-
-    <tr>
-      <th><%= f.label(:city, "* City:") %></th>
-      <td><%= f.text_field :city, :size => 30 %></td>
-    </tr>
-
-    <tr>
-      <th><%= f.label(:province, "* Province/State:") %></th>
-      <td><%= f.text_field :province, :size => 30 %></td>
-    </tr>
-
-    <tr>
-      <th><%= f.label(:country, "* Country:") %></th>
-      <td><%= f.text_field :country, :size => 30 %></td>
-    </tr>
-
-    <tr>
-      <th><%= f.label(:postal_code, "Postal/ZIP Code:") %></th>
-      <td><%= f.text_field :postal_code, :size => 8 %></td>
-    </tr>
-
-    <tr>
-      <th><%= f.label(:time_zone, "Time Zone:") %></th>
-      <td>
-        <%= f.time_zone_select :time_zone,
-            ActiveSupport::TimeZone.all.select { |t| t.name =~ /canada/i },
-            { :default => ActiveSupport::TimeZone['Eastern Time (US & Canada)'], :include_blank => true }
+  <div class="generalbox">
+
+    <p class="warning"> Fields with an asterisk (*) are mandatory.</p>
+
+    <fieldset class="groupentry">
+
+      <legend>
+        Personal Info
+      </legend>
+
+      <div class="tree-column-grid">
+
+        <%= f.label(:title, "Title:") %>
+        <%= f.text_field :title %> <span class="field_note">for example: 'Mrs.', 'Mr', 'Dr.', etc.</span>
+
+
+        <%= f.label(:first, "* First Name:") %>
+        <%= f.text_field :first %>
+        <br>
+
+        <%= f.label(:middle, "Middle Name:") %>
+        <%= f.text_field :middle %>
+        <br>
+
+        <%= f.label(:last, "* Last Name:") %>
+        <%= f.text_field :last %>
+        <br>
+
+        <%= f.label(:login, "Preferred 'login' name:") %>
+        <%= f.text_field :login %>
+
+
+
+        <span class="field_note">
```

Comment:
the class `field_note` only exists in the neurohub css; in cbrain it's `field_explanation`.